content/platform/alert-runbooks.md

```diff
@@ -29,6 +29,22 @@ Each alert should contain a short description and a deep link to the correspondi
 
 2. Is the Cloud NAT configured correctly?
 
+##### KubePodCannotConnectToDeveloperPortal
+
+1. Is the developer portal running?
+   ```
+   kubectl -n developer-portal get all
+   ```
+
+2. Are other alerts such as `KubePodCannotConnectToInternet` or `ContainerInErrorState` firing?
```

Comment:
would add here a troubleshooting link with a command on how to test connectivity between internet/pod